src/cascadia/TerminalSettingsModel/TerminalSettingsModel.vcxproj

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{CA5CAD1A-d7ec-4107-b7c6-79cb77ae2907}</ProjectGuid>
+    <ProjectName>TerminalSettingsModel</ProjectName>

```

Comment:
can you change something so that this goes into a DLL called `Microsoft.Terminal.Settings.Model.dll`?